assets/base/config.json

```diff
@@ -41,5 +41,7 @@
 
     "AppGroupId": "group.com.mattermost.rnbeta",
 
-    "ShowSentryDebugOptions": false
+    "ShowSentryDebugOptions": false,
+
+    "CustomRequestHeaders": null
```

Comment:
If we default this to `{}`, it might make it clear to whoever uses it that it can take any number of key/value pairs